### DISCUSSION TITLE
How do you make a revision into a published record?

### DISCUSSION THREAD

**Discussion by SeriousKen** [oddvalue/laravel-drafts#38] [Q&A]
I'm trying to figure out how to turn a revision, particularly one that isn't current, into a published one but something is going wrong. I've used `setLive()` on the model followed by `save()` which works to publish the record, but I now have two records marked as current. This usually happens when `is_published` and `is_current` are true for different records and the one I want to make published is not current.

**Comment by oddvalue** [comment]
This was a bug, it's fixed in #39
